concent_api/requirements.lock

```diff
@@ -15,13 +15,14 @@ cffi==1.11.5
 chardet==3.0.4
 cloudpickle==0.5.3
 coincurve==7.1.0
+construct==2.9.45
 coverage==4.5.1
 cycler==0.10.0
 cytoolz==0.9.0.1
 dask==0.18.1
 decorator==4.3.0
 Django==1.11.13
-django-constance[database]==2.2.0
+django-constance==2.2.0
```

Comment:
Why is this being changed?

Reply:
I asked Karol about it, in the past it was necessary because of problems with database. Now everything is OK and it is not required any more

Reply:
I know but I'm asking why it is being changed in this pull request. We added a script for generating `requirements.lock` always in the canonical format (#711) - why did its output suddenly change now and not back then?

Reply:
>why did its output suddenly change now and not back then?

Because back then I have only added script to generate `requirements.lock`. I have not generate new `requirements.lock` file in this -> https://github.com/golemfactory/concent/pull/754 PR.

Reply:
OK. But in that case there should be a separate commit that does this (with its description saying what you said above) so that it's clear that it's just cleanup of a previous situation and not something that's a part of the solution.